platform/viewer/src/connectedComponents/ConnectedViewer.js

```diff
@@ -1,13 +1,20 @@
-import { connect } from "react-redux";
-import Viewer from "./Viewer.js";
-import OHIF from "@ohif/core";
+import { connect } from 'react-redux';
+import Viewer from './Viewer.js';
+import OHIF from '@ohif/core';
 
 const { setTimepoints, setMeasurements } = OHIF.redux.actions;
 
-const mapStateToProps = (state, ownProps) => {
+const getActiveServer = servers => {
+  const isActive = a => a.active === true;
+  return servers.servers.find(isActive);
+};
```

Comment:
Servers should now be exposed via `AppContext`. I wonder if it's worth having a `getActiveServer` method for it, or leveraging an equivalent selector 🤔 (not blocking)